### PR TITLE
Add block fetching for stellar

### PR DESCRIPTION
**Pull request by madcake**

### CODE REVIEW

File: platform/stellar/client.go [hunk 1]
```diff
@@ -31,3 +34,49 @@ func (c *Client) GetTxsOfAddress(address string) (txs []Payment, err error) {
...
+
+	return ledgers.Embedded.Records[0].Sequence, nil
+}
+
+func (c *Client) GetBlockByNumber(num int64) (block *Block, err error) {
+	ledger, err := c.getLedger(num)
+	if err != nil {
+		return nil, err
+	}
+	path := fmt.Sprintf("%s/ledgers/%s/payments?limit=100&order=desc", c.API, num)
```
Comment (terorie) [inline platform/stellar/client.go]: Wrong `Sprintf` usage

File: platform/stellar/client.go [hunk 2]
```diff
@@ -31,3 +34,49 @@ func (c *Client) GetTxsOfAddress(address string) (txs []Payment, err error) {
...
+	payments, err := c.getTxs(path)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Block{Ledger: *ledger, Payments: payments}, nil
+}
+
+func (c *Client) getLedger(num int64) (ledger *Ledger, err error) {
+	path := fmt.Sprintf("%s/ledgers/%s", c.API, num)
```
Comment (terorie) [inline platform/stellar/client.go]: Wrong `Sprintf` usage

**Comment by puripong36575** [comment]
hi ok perfect

File: platform/stellar/client.go [hunk 3]
```diff
@@ -31,3 +34,49 @@ func (c *Client) GetTxsOfAddress(address string) (txs []Payment, err error) {
...
+
+	return ledgers.Embedded.Records[0].Sequence, nil
+}
+
+func (c *Client) GetBlockByNumber(num int64) (block *Block, err error) {
+	ledger, err := c.getLedger(num)
+	if err != nil {
+		return nil, err
+	}
+	path := fmt.Sprintf("%s/ledgers/%d/payments?limit=100&order=desc", c.API, num)
```
Comment (vikmeup) [inline platform/stellar/client.go]: `limit=100` should we increase this number?

**Pull request merged**